app/services/region_backfill.rb

```diff
@@ -89,13 +89,8 @@ def find_or_create_region_from(parent:, region_type:, name: nil, source: nil)
     existing_region = parent.children.find_by(name: region_name, region_type: region_type)
     return existing_region if existing_region
 
-    region = DryRunRegion.new(Region.new(name: region_name, region_type: region_type), dry_run: dry_run?, logger: logger)
-    if source
-      region.slug = source.slug
-      region.source = source
-    end
-    region.path = [parent.path, region.path_label].compact.join(".")
-
+    region = DryRunRegion.new(Region.new(name: region_name, region_type: region_type, reparent_to: parent), dry_run: dry_run?, logger: logger)
```

Comment:
`reparent_to` now handles setting the path for us

Reply:
We also had something similar here, but removed it again, because it seemed like introducing a new API for re-parenting was confusing and not obvious. I'm open to this, but reluctantly so.